SwiftCompilerSources/force_lib.c

```diff
@@ -0,0 +1,14 @@
+//===--- force_lib.c ------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Dummy source file to force CMake generated SwiftInTheCompiler.xcodeproj to sucessfully build static
+/// libraries containing only object fies for "bootstrap" process building Swift sources into compiler.
```

Comment:
... files ...

Reply:
Also ensure that doesn't overflow the 80-character limit for a line.

Reply:
Done.